client/reader/_style.scss

```diff
@@ -238,20 +238,24 @@
 	padding: 0;
 	vertical-align: middle;
 }
+
 .tiled-gallery-caption { /* Captions */
 	background: #eee;
 	background: rgba( 255,255,255,0.8 );
 	color: #333;
 	font-size: 13px;
 	font-weight: 400;
 	overflow: hidden;
-	padding: 10px 0;
+	padding: 10px 20px;
 	position: absolute;
 		bottom: 0;
-	text-indent: 10px;
 	text-overflow: ellipsis;
 	width: 100%;
-	white-space: nowrap;
+  transition: opacity .25s ease-in-out;
+  opacity: 0;
```

Comment:
The spacing introduced here is a little messed up. Your editor might have inserted spaces instead of tabs.